lib/core-integration/src/Test/Integration/Scenario/CLI/Keys.hs

```diff
@@ -74,17 +74,21 @@ spec  = do
               \e99a00cfed56e9516bc947f327a73e0849882a32a682932c51b42156055abb0b\
               \5d3661deb9064f2d0e03fe85d68070b2fe33b4916059658e28ac7f7f91ca4b12"
 
-        it "bryon keys fail" $ do
-            keyStderr ["child", "--path", "0"] byronKey
-                `shouldReturn`
-                "That extended private key looks \
-                \weird. Is it encrypted? Or is it an old Byron key?\n"
-
-        it "encrypted keys fail" $ do
-            keyStderr ["child", "--path", "0"] encryptedKey
-            `shouldReturn`
-                "That extended private key looks \
-                \weird. Is it encrypted? Or is it an old Byron key?\n"
+        it "byron keys does not fail anymore" $ do
+            res <- key ["child", "--path", "0"] byronKey
+            res `shouldBe`
+                "460d95f55f8940c7145ab251da03f6a8767f2306b1f5345afd10090a558515\
+                \0abb4902024903aa130d0ea3524968539071c0708a020c8181c913b9c7f306\
+                \3b209bcb6f8af16317702127ab4169d2d7f11807d464b2f427b3ca45d5d9f0\
+                \578288\n"
+
+        it "encrypted byron keys does not fail anymore" $ do
+            res <- key ["child", "--path", "0"] encryptedKey
+            res `shouldBe`
+                "7d0ef39b2f7de32e8ca24cfe7a9cf1663a860778c42c70ef41ad851fa6606c\
+                \bbcfee7fbacf0e7141e449aa18e8d0c7206c0b7a71d3129751b516bf0a2abb\
+                \2383ca7b962e3bc5aa7d049f33606c8851a7ba94432ef335e47d77575e2fce\
+                \a9813c\n"
```

Comment:
Why though? I'd still expect these to fail.

Reply:
I think it used to fail because the encrypted keys didn't (appear) to have the correct tweaked bits.

Ignoring the tweak, there's no way the tell that the key is encrypted. (I think)

Reply:
> there's no way the tell that the key is encrypted. 

We can try changing its passphrase to `""`. If the key was indeed non encrypted, this should be a no-op :thinking:

Reply:
> We can try changing its passphrase to "".

Not sure how that works, but keep in mind that the 96-byte format — unlike a `XPrv` — doesn't contain the public key.